src/media/css/pretty-select.styl

```diff
@@ -52,3 +52,11 @@ $select-height = 30px;
         padding: 5px;
     }
 }
+
+.app-list-filters .pretty-select {
```

Comment:
i think this should go where the app-list-filter styles are. yeah, it's a fine line on where to put page vs component rules